client/network/src/lib.rs

```diff
@@ -89,13 +89,14 @@ impl BlockAnnounceData {
 	///
 	/// This will not check the signature, for this you should use [`BlockAnnounceData::check_signature`].
 	fn validate(&self, encoded_header: Vec<u8>) -> Result<(), Validation> {
-		let candidate_hash =
-			if let CompactStatement::Seconded(h) = self.statement.unchecked_payload() {
-				h
-			} else {
-				tracing::debug!(target: LOG_TARGET, "`CompactStatement` isn't the candidate variant!",);
-				return Err(Validation::Failure { disconnect: true })
-			};
+		let candidate_hash = if let CompactStatement::Seconded(h) =
```

Comment:
this is `cargo +nightly fmt` change, not related to this PR

Reply:
CI is failing in this file. I suggest to roll back those changes.

Reply:
CI is failing in this file. I suggest to roll back those changes.